File: app/src/main/java/com/dx/demi/archdiary/activity/MainActivity.java
package com.dx.demi.archdiary.activity;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.Toolbar;

import com.dx.demi.archdiary.R;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);// 将 Toolbar 的实例传入
    }
}

File: app/src/main/java/com/dx/demi/archdiary/activity/MainActivity.kt
package com.dx.demi.archdiary.activity

import android.support.v7.app.AppCompatActivity
import android.os.Bundle

import com.dx.demi.archdiary.R
import kotlinx.android.synthetic.main.activity_projects.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_projects)
        setSupportActionBar(toolbar)// 将 Toolbar 的实例传入
    }
}
